includes/Core/Storage/User_Options.php

```diff
@@ -122,6 +122,17 @@ public function delete( $option ) {
 		return (bool) delete_user_option( $this->user_id, $option );
 	}
 
+	/**
+	 * Gets the ID of the user that options are controlled for.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return int User ID.
+	 */
+	public function get_user_id() {
+		return $this->user_id;
+	}
```

Comment:
Do you think this is worth adding a test for?

Reply:
Sounds good, will add one

Reply:
Sounds good, will add one